app/src/main/java/com/mealsmadeeasy/ui/home/glance/WeekAtAGlanceMealViewHolder.kt

```diff
@@ -58,4 +64,40 @@ class WeekAtAGlanceMealViewHolder(
         })
         popup.show()
     }
+
+    private fun onClickEditServings() {
+        val root = View.inflate(itemView.context, R.layout.view_edit_servings, null)
+        val servingsText = root.findViewById<TextView>(R.id.add_to_plan_servings)
+        val plusButton = root.findViewById<ImageView>(R.id.add_to_plan_plus)
+        val minusButton = root.findViewById<ImageView>(R.id.add_to_plan_minus)
+
+        var numServings = mealPortion.servings
+        servingsText.text = numServings.toString()
+        minusButton.isEnabled = numServings != 1
+
+        minusButton.setOnClickListener {
+            if (numServings != 1) {
+                numServings--
+                servingsText.text = numServings.toString()
+                minusButton.isEnabled = (numServings > 1)
+            }
+        }
+
+        plusButton.setOnClickListener {
+            numServings++
+            minusButton.isEnabled = (numServings > 1)
+            servingsText.text = numServings.toString()
+        }
+
+
+        AlertDialog.Builder(itemView.context)
+                .setView(root)
+                .setPositiveButton("Confirm") { _, _ ->
+                    onUpdateMeal(MealPortion(mealPortion.meal, numServings))
+                }
+                .setNegativeButton("Cancel") { _, _ ->
+                    // Do nothing.
+                }
+                .show()
+    }
```

Comment:
Do you care about persisting this dialog across configuration changes? 'Cause this dialog will disappear if a user rotates their phone.